packages/api-client/.nycrc.json

```diff
@@ -3,6 +3,6 @@
   "exclude-after-remap": false,
   "statements": 92.0,
   "functions": 88.0,
-  "branches": 29.0,
+  "branches": 28.0,
```

Comment:
The metrics on nyc aren't right for branch coverage for nearly all of our API tests - I believe this has to do with how we're mocking out the API calls, but isn't directly related to this work other than adding an additional method tested similarly to the rest of the package.  

I've entered a ticket to look into the coverage metrics and address *or* write an ADR re: this metric in the form of CUMULUS-3775.